docs/src/main/paradox/buildtools/index.md

```diff
@@ -0,0 +1,11 @@
+# Build Tool Support
+
+@@toc
+
+@@@index
+ 
+ * [SBT](sbt.md)
```

Comment:
`Sbt` or `sbt`, we stopped writing it as "SBT" years ago AFAIR -- cc @eed3si9n

Reply:
I always write it as sbt, all lower-cased.

Reply:
I will fix it but it is an index label which isn't used anyway ;)